ext/hal/nxp/mcux/README

```diff
@@ -17,6 +17,7 @@ Status:
    ----------------------------------------------------------------------------
    MIMXRT1015	SDK 2.5.0 (2019-03-11)		REL_2.5.0_RT1015_RFP
    LPC54114	SDK 2.5.0 (2018-12-17)		REL_2.5.0_REL9_RFP_RC3_7_1
+   LPC55S69	SDK 2.5.1 (2019-04-18)		REL_SDK_NIOBE4_2.5.1_RFP_RC1
```

Comment:
Per https://github.com/zephyrproject-rtos/zephyr/pull/16049#discussion_r283954331, please use the "revision" attribute instead of the tag.

Reply:
@MaureenHelm done!